data/microos/butane/script

```diff
@@ -97,6 +97,8 @@ echo 'FONT=eurlatgr.psfu' >> /etc/vconsole.conf
 if ! grep -q tumbleweed /etc/os-release; then
     create_fs
     systemd_mount
+else
+    mount /home
```

Comment:
Isn't `/home` already available in a linux standard?

Reply:
the changes made by combustion script seemed to be over-mounted. If combustion is used outside of TW, mounting /home again produced the expected changes.

Reply:
Very strange and not clear to me: it may be _combustion_ should be fixed before, instead. 
But ok.